repos/system_upgrade/el7toel8/models/xfspresence.py

```diff
@@ -3,7 +3,14 @@
 
 
 class XFSPresence(Model):
+    """ A generic message indicating if XFS was found and if any parition has ftype = 0 or not """
     topic = SystemInfoTopic
 
     present = fields.Boolean(default=False)
+    """ XFS is used on this partition """
```

Comment:
This model seems to be inconsistent to me.

> XFS is used on this partition

which partition?

> List of mountpoints that have ftype=0

Previously, it mentioned one partition but now we are talking about list of mount points

Maybe I am just misunderstanding it, could you please elaborate on this model?